### DISCUSSION TITLE
Import data in table with link

### DISCUSSION THREAD

**Discussion by syji35** [nocodb/nocodb#3601] [Q&A]
Hi,

I built two "country" and "city" on my nocodb (version 0.92.4)
I have a "city" table with a a "linkAnotherRecord" to table "country".

![image](https://user-images.githubusercontent.com/45880189/189846888-9f745bf7-e3e6-4c17-a00b-a8a2c5eb1df5.png)
![image](https://user-images.githubusercontent.com/45880189/189846979-ef94f3dd-a4a1-467d-b4cf-e0609c9db893.png)

When I import a csv file to fill the "City" table, the link with the "Country" table is not made.

Is this normal?

Exemple of my csv :  

![image](https://user-images.githubusercontent.com/45880189/189848419-69c02ab7-e0bd-4036-ab12-5a0ac6bd1393.png)

Result when I import this :  

![image](https://user-images.githubusercontent.com/45880189/189847745-7aff90e7-1045-4dd4-ab98-619c5cca5896.png)
![image](https://user-images.githubusercontent.com/45880189/189847815-4e7402c5-9e79-47d5-80be-bc8a21753bbc.png)

**Comment by o1lab** [comment]
Internally within the lib uploading csv to related table fields is not possible. Hence the feature is said to be in alpha.

@wingkwong : pls see we dont show the related field names when we upload. 

We will improve this going forward.
